pallets/parachain-staking/src/benchmarking.rs

```diff
@@ -44,7 +44,7 @@ fn setup_collator_candidates<T: Config>(
 	let collators: Vec<T::AccountId> = (current_collator_count..num_candidates)
 		.map(|i| account("collator", i.saturated_into::<u32>(), COLLATOR_ACCOUNT_SEED))
 		.collect();
-	let amount: T::CurrencyBalance = default_amount.unwrap_or_else(T::MinCollatorCandidateStake::get);
```

Comment:
I wonder why this line (`T::MinCollatorCandidateStake::get`) was not caught 🥲

Reply:
should be the same? Actually clippy should complain now?

Reply:
> should be the same? Actually clippy should complain now?

Why should clippy complain now? And shouldn't `T:MinCollatorCandidateStake:get` without the parenthesis NOT work?

Reply:
As discussed: Reverted in 0cb2746